addons/l10n_account_edi_ubl_cii_tests/tests/test_xml_ubl_nl.py

```diff
@@ -198,6 +209,34 @@ def test_export_import_refund(self):
         self.assertEqual(xml_filename[-10:], "nlcius.xml")
         self._assert_imported_invoice_from_etree(refund, xml_etree, xml_filename)
 
+    def test_export_fixed_tax_nlcius_and_peppol(self):
+        """
+        Ensure that an invoice containing a product with a fixed tax posted to a journal with the peppol and nlcius edi
+            tags generates edi documents with accurate LineExtensionAmount values
+        """
+        self.journal.edi_format_ids = [
+            Command.link(self.env.ref('account_edi_ubl_cii.ubl_bis3').id)
+        ]
+
+        self.tax_10_fixed.sequence = 1
```

Comment:
this tax already has a sequence = 1 ... :grimacing: 
The sequence of the fixed taxed should be **lower** than the sequence of the percent tax.
You can check by yourself: the `invoice.amount_total` should be (50 + 10) * 1.07